lib/auth/securityQuestions.ts

```diff
@@ -90,7 +90,7 @@ export async function createSecurityAnswers(
 ): Promise<void> {
   checkPrivileges(ability, [
     {
-      action: "create",
+      action: "update",
```

Comment:
Needed to change to `update` because it is tied to the User object which no one can `create`.